tritium-lib/src/main/java/com/palantir/tritium/Tritium.java

```diff
@@ -35,12 +36,27 @@ private Tritium() {
      *
      * @param serviceInterface service interface
      * @param delegate delegate to instrument
-     * @param metricRegistry metric registry
+     * @param metrics metric registry
      * @return instrumented proxy implementing specified service interface
+     * @deprecated use {@link #instrument(Class, Object, TaggedMetricRegistry)}
      */
-    public static <T, U extends T> T instrument(Class<T> serviceInterface, U delegate, MetricRegistry metricRegistry) {
+    @Deprecated
+    public static <T, U extends T> T instrument(Class<T> serviceInterface, U delegate, MetricRegistry metrics) {
+        return instrument(serviceInterface, delegate, TaggedMetricRegistry.getDefault());
```

Comment:
release note: This maintains back compatibility, but we should move consumers over to the method below to inject the desired tagged metric registry.

Reply:
per comment above, kind of think maybe we should actually leave the old metrics in place until people convert